File: Ejercicio1.c
//Ingresar 10 valores por teclado. Presentar la suma y promedio
//Hecho por sebastian Gómez
#include <stdio.h>
#include <time.h>
int main(){
 int a, suma=0,i;
 double promedio, tiempoTotal;
 clock_t inicio, fin;
    
 inicio= clock();
 for ( i=1 ; i<=10; i++) {
  printf("Ingrese el numero %d:", i);
  scanf("%d", &a);
  suma= suma+a;}
  promedio=(double)suma/i;
 printf("El total de la suma es: %d\n", suma);
 printf("El promedio total es: %.2f\n", promedio);
 fin=clock();
 tiempoTotal = (double)(fin - inicio) / CLOCKS_PER_SEC;
 printf("Tiempo de respuesta del programa: %f segundos\n", tiempoTotal);
 return 0;
}

File: Ejercicio1.py
# Ingresar 10 valores por teclado. Presentar la suma y promedio
import time
inicio=time.time()
suma=0
for i in range (1,11):
    print("Ingrese el valor número",i)
    a= int(input())
    suma= suma+a
    promedio=suma/i    

print("El total de la suma de los ",i," números ingresados es: ",suma)
print("El promedio total de los ",i," números ingresados es: ",promedio)
fin=time.time ()
tiempoTotal=fin-inicio
print(f"\nTiempo de ejecución del programa : {tiempoTotal:.6f} segundos")